db/migrate/20170728140134_remove_email_from_enterprises.rb

```diff
@@ -0,0 +1,21 @@
+class RemoveEmailFromEnterprises < ActiveRecord::Migration
+  def up
+    Enterprise.select([:id, :email]).each do |e|
+      contact_user = Spree::User.find_by_email e.email
+      manager = EnterpriseRole.find_or_initialize_by_user_id_and_enterprise_id(contact_user.id, e.id)
```

Comment:
This line throws an error in my merged branch:
```
Called id for nil, which would mistakenly be 8 -- if you really wanted the id of nil, use object_id
```

It looks like `contact_user` can be `nil`.

Reply:
So it looks like we'll need something for the cases where the current email that receives notifications doesn't correspond to a registered OFN user. 

Seeing how it will need to be a Spree::User from now on, I guess we'll have to add something into the migration that adds a new user record for that emails if they don't already exist and send them a confirmation email? Which sounds like the "invite a manager" functionality discussed in #1593. Maybe we'll have to add this and call it from the migration?

Any thoughts? @mkllnk @sauloperez @enricostano

Reply:
Talked offline. TL;DR:
yes... we need another data migration that creates users + enterprise roles

Reply:
@mkllnk I can't figure out how `contact_user` can be `nil`. Any time the `Enterprise.email` field is changed and it doesn't correspond to an existing user, a new user gets created with that email address and the `Enterprise.email` field doesn't update until they confirm...?

Reply:
I'm assuming any enterprises who's email doesn't correspond to a user is due to legacy data, and probably in enterprises that are no longer in use?

Reply:
I've added a quick change to the migration and tested it by manually adding an enterprise email that didn't have a user record associated with it.